terraform/main.tf

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+resource "juju_application" "gnb" {
+  name  = var.app_name
+  model = var.model_name
+
+  charm {
+    name    = "sdcore-gnb-integrator"
+    channel = var.channel
+  }
+  config = var.config
+  units  = 1
+  trust  = true
```

Comment:
We don't need the `trust` parameter here. It's only required when the charm is talking to K8s API.